service-base/src/main/java/org/eclipse/hono/service/AbstractApplication.java

```diff
@@ -148,6 +143,17 @@ public final void startup() {
         }
     }
 
+    private CompositeFuture deployServices(int maxInstances) {
+        final List<Future> deploymentTracker = new ArrayList<>();
+        
+        for (ObjectFactory<? extends AbstractServiceBase> serviceFactory : serviceFactories) {
+            final AbstractServiceBase firstServiceInstance = serviceFactory.getObject();
```

Comment:
creating the first verticle here only does not seem to work when we deploy the verticles in a `@PostCreate` method.

however, I think I have fixed the problem by instead using `ApplicationRunner.run()` to deploy the verticles (ab58e424c44f753f6ea23bde7c8afa729ae3d71c). Can you please rebase?

I also think that this method should be merged into *deployServiceVerticles*.